app/views/idv/doc_auth/verify.html.erb

```diff
@@ -40,7 +40,11 @@
       class: 'btn btn-link',
     ) %>
   </div>
-  <%= "#{t('doc_auth.forms.ssn')}: #{flow_session[:pii_from_doc][:ssn]}" %>
+  <%= t('doc_auth.forms.ssn') %>
+  <div class='sm-col sm-col-8 px0 mt0 mb3'>
+    <input value="<%= flow_session[:pii_from_doc][:ssn] %>" class="block col-12 field password disabled ssn ssn-toggle" aria-label="<%= t('doc_auth.forms.ssn') %>" aria-invalid="false" aria-required="false" readonly maxlength="11" pattern="^\d{3}-?\d{2}-?\d{4}$" size="11" type="password" name="doc_auth[ssn]" id="doc_auth_ssn" />
```

Comment:
WDYT of using a tag helper so we don't have to manually escape the `value` or `aria-label` attributes?
Also some line breaks? 😁 
```suggestion
    tag.input value: flow_session[:pii_from_doc][:ssn],
              class: 'block col-12 field password disabled ssn ssn-toggle'
              aria: { label: t('doc_auth.forms.ssn'), invalid: false, required: false },
              readonly: true,
              maxlenth: 11,
              pattern: "^\d{3}-?\d{2}-?\d{4}$",
              size: "11",
              type: "password",
              name: "doc_auth[ssn]",
              id: "doc_auth_ssn"
           
```